client/selectors/getAppsForConnection.js

```diff
@@ -1,14 +1,24 @@
-import { createSelector } from 'reselect';
+export default function normalizeErrorMiddleware() {
```

Comment:
What happened here? Why is all of the code replaced with something totally different?